app/services/migration/migrators/statement_item.rb

```diff
@@ -11,6 +11,8 @@ def model
 
       def ecf_statement_items
         Migration::Ecf::Finance::StatementLineItem
+          .joins(:statement)
+          .where("statements.type ILIKE 'Finance::Statement::NPQ%'")
```

Comment:
maybe we go with `ParticipantDeclaration` types as they are always either `"ParticipantDeclaration::NPQ"`or `"ParticipantDeclaration::ECF"` 🤔